### DISCUSSION TITLE
Favorite views issue

### DISCUSSION THREAD

**Discussion by Nafhaa** [serenity-is/Serenity#6033] [Help] [answered]
When favorite views are enabled getting following error,  I have no longer access to issues on StartSharp so posting here,  needed to recreate the discussion as previous discussion was locked and cant add comments further.

![137096888-289c1fd2-6637-44ed-9dea-42991639bd30](https://user-images.githubusercontent.com/58759165/137184751-c623a662-6c6a-4efd-8cab-e50796347186.png)

Code used:
new Serenity.FavoriteViewsMixin({
grid: this,
});

**Comment by edwardch** [comment]
Duplicate of [6031](https://github.com/serenity-is/Serenity/discussions/6031)

**Reply by Nafhaa** [reply]
That discussion is been deleted if you check

**Reply by edwardch** [reply]
It looks like it has been deleted now.

Your issue is related to the premium template so you should rather post it there if you have a license.

**Reply by VictorTomaili** [reply]
Currently, it's covered by community support. It's okay.

**Comment by Nafhaa** [comment, marked as answer]
This issue is been resolved.
As my project was designed in Serenity community edition(in 2016) and then we migrated to StartShap last year where found that the folder name was UserPreference and in new it was UserPrefrences. Due to which the include was not working, adding details so someone may found this useful
